build.gradle

```diff
@@ -248,8 +240,7 @@ test {
     minHeapSize = "1G"
     maxHeapSize = "4G"
 
-
-    if (CI == "true" && CLOUD == "false" ) {
+    if (TEST_VERBOSITY == "minimal") {
```

Comment:
This toggle is between showing all test output and showing abbreviated test output.  Previously we showed abbreviated test output when running on travis and NOT running cloud tests, because there were only a few cloud tests so there wasn't any point in minimizing their output and it was more useful to be able to see them as they ran.  I figured it would make more sense to set the test output verbosity explicitly instead of having it depend on some combination of other confusing factors.  I'm not sure that TEST_VERBOSTY = minimal is the right UI choice though, any suggestion on names?  `ABBREVIATE_TEST_OUTPUT = true`?